tests/test_helper.py

```diff
@@ -28,3 +28,55 @@ def test_get_prep_value(self):
         field = fields.IntegerQuantityField("meter")
         with self.assertRaises(ValueError):
             field.get_prep_value("foobar")
+
+
+class TestContextHandling(TestCase):
+    def setUp(self):
```

Comment:
I know the other lack it as well but could you add some doc string explaining why the test are needed?
May also refer to the isseue?

Reply:
Done